logging/unit_tests/test_client.py

```diff
@@ -571,6 +571,59 @@ def test_list_metrics_with_paging(self):
             },
         })
 
+    def test_get_default_handler(self):
+        import os
+        from google.cloud._testing import _Monkey
+        from google.cloud.logging.handlers.GAEHandler import GAEHandler
+        from google.cloud.logging.handlers.GKEHandler import GKEHandler
+        from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+
+        client = self._makeOne(
+            project=self.PROJECT, credentials=_Credentials(),
+            use_gax=False)
+        client.get_default_handler()
+
+        client = self._makeOne(self.PROJECT, credentials=_Credentials())
+
+        with _Monkey(os, getenv=lambda var: var == 'GAE_APPENGINE_HOSTNAME'):
+            handler = client.get_default_handler()
+
+        self.assertIsInstance(handler, GAEHandler)
+
+        with _Monkey(os, getenv=lambda var: var == 'KUBERNETES_SERVICE'):
+            handler = client.get_default_handler()
+
+        self.assertIsInstance(handler, GKEHandler)
+
+        with _Monkey(os, getenv=lambda _: None):
+            handler = client.get_default_handler()
+
+        self.assertIsInstance(handler, CloudLoggingHandler)
+
+    def test_setup_logging(self):
+        from google.cloud._testing import _Monkey
+        import google.cloud.logging.client as MUT
+        client = self._makeOne(
+            project=self.PROJECT, credentials=_Credentials(),
+            use_gax=False)
+        client.get_default_handler()
+
+        setup_logging_mock = _SetupLogging()
+        with _Monkey(MUT, setup_logging=setup_logging_mock):
+            client.setup_logging()
+
+        self.assertTrue(setup_logging_mock.called)
+
+
+class _SetupLogging(object):
+
+    def __init__(self):
+        self.called = False
+
+    def __call__(self, _, log_level,
+                 excluded_loggers=None):  # pylintdisable=unused-variable
```

Comment:
Ditch the disable and just use the variable (e.g. `self._excluded = excluded_loggers`) and use a real name instead of `_` in `self, _, log_level`